ovos_PHAL_plugin_wallpaper_manager/__init__.py

```
@@ -271,39 +238,38 @@ def get_wallpaper_collection_from_provider(self, message):
-            try:
-                set_wallpaper(wallpaper)
-            except:
-                # https://github.com/OpenVoiceOS/ovos-PHAL-plugin-wallpaper-manager/issues/7
-                pass  # TODO - happens in EGLFS, fix later
+        # if running on a desktop, also change it's wallpaper
+        # TODO - config flag?
+        try:
+            set_wallpaper(wallpaper)
+        except:
+            # https://github.com/OpenVoiceOS/ovos-PHAL-plugin-wallpaper-manager/issues/7
+            pass  # TODO - happens in EGLFS, fix later
```

Ruff

Use `contextlib.suppress(Exception)` instead of `try`-`except`-`pass`

(SIM105)